projects/charts-ng/src/components/si-chart-cartesian/si-chart-cartesian.component.ts

```diff
@@ -29,10 +29,14 @@ import {
   styleUrl: '../si-chart/si-chart.component.scss'
 })
 export class SiChartCartesianComponent extends SiChartComponent implements OnChanges {
+  /** The series data for the chart. */
```

Comment:
Lets just say series, because the data would mean something different.

```suggestion
  /** The series for the chart. */
```